tests/slow/src/test/scala/tests/pants/PantsSlowSuite.scala

```diff
@@ -0,0 +1,254 @@
+package tests.pants
+
+import java.io.FileInputStream
+import java.nio.charset.StandardCharsets
+import scala.meta.internal.builds.{PantsBuildTool, PantsDigest}
+import scala.meta.internal.io.InputStreamIO
+import scala.meta.internal.metals.Messages._
+import scala.meta.internal.metals.UserConfiguration
+import scala.meta.io.AbsolutePath
+import tests.BaseImportSuite
+import scala.meta.internal.builds.BuildTool
+
+object PantsSlowSuite extends BaseImportSuite("pants") {
+
+  val buildTool = PantsBuildTool(() => userConfig)
+
+  private def preInitialized = {
+    val pants_targets_config =
+      s"""
+         |{
+         |  "pants-targets": "src::"
+         |}
+         |""".stripMargin
+    server.didChangeConfiguration(pants_targets_config)
+  }
+
+  override def utestBeforeEach(path: Seq[String]): Unit = {
+    super.utestBeforeEach(path)
+    installPants()
+  }
+
+  def installPants(): Unit = {
+    cleanWorkspace()
+    val pants = BuildTool.copyFromResource(workspace.toNIO, "pants")
+    pants.toFile().setExecutable(true)
+    import scala.sys.process._
+    val exit = List(pants.toString(), "generate-pants-ini").!
+    require(exit == 0, "failed to generate pants.ini")
```

Comment:
Can we have a static `pants.ini` defined with rest of the files. This will make the tests run much slower.